app/views.py

```diff
@@ -0,0 +1,160 @@
+    return HttpResponseRedirect('/')
+
+
+def logout_page(request):
+    logout(request)
+    return HttpResponseRedirect('/')
+
+
+@login_required
+def home(request):
+    return HttpResponseRedirect('/')
+
+
+class MyProfileView(TemplateView):
+    """
+    Enables viewing of the user's profile in the HTML site, by the profile
+    owner.
+    """
+    template_name = 'app/viewprofile.html'
+
+    def get_context_data(self, **kwargs):
+        """ Retrieve the user profile
+        """
+        context = super(MyProfileView, self).get_context_data(**kwargs)
+        user = self.request.user
+        profile = user.profile
+        context['username'] = user.username
+
+        if profile.alias:
+            context['alias'] = profile.alias
+        else:
+            context['alias'] = 'Anonymous'
+
+        context['date_of_birth'] = profile.date_of_birth
+        return context
+
+
+class MyProfileEdit(FormView):
+    """
+    Enables editing of the user's profile in the HTML site
+    """
+    form_class = EditProfileForm
+    template_name = 'app/editprofile.html'
+
+    def get_initial(self):
+        initial = self.initial.copy()
+        user = self.request.user
+        profile = user.profile
+        initial['username'] = user.username
+
+        if profile.alias:
+            initial['alias'] = profile.alias
+        else:
+            initial['alias'] = 'Anonymous'
+
+        initial['date_of_birth'] = profile.date_of_birth
+        return initial
+
+    def get_form(self, form_class):
+        form = super(MyProfileEdit, self).get_form(form_class)
+        return form
+
+    def form_valid(self, form):
+        user = self.request.user
+        profile = user.profile
+        profile.alias = form.cleaned_data['alias']
+        profile.save()
+        return HttpResponseRedirect(reverse('view_my_profile'))
+
+
+class ProfilePasswordChangeView(FormView):
+    form_class = ProfilePasswordChangeForm
+    template_name = 'app/change_password.html'
+
+    def get_form(self, form_class):
+        form = super(ProfilePasswordChangeView, self).get_form(form_class)
+        return form
+
+    def form_valid(self, form):
+        user = self.request.user
+        if user.check_password(form.cleaned_data['old_password']):
+            user.set_password(form.cleaned_data['new_password'])
+            user.save()
+            return HttpResponseRedirect(reverse('view_my_profile'))
+        else:
+            messages.error(
+                self.request,
+                'The Old password is incorrect.'
+            )
+            variables = RequestContext(
+                self.request,
+                {'form': form}
+            )
+            return render_to_response(
+                'app/change_password.html',
+                variables,
+            )
+        variables = RequestContext(self.request, {'form': form})
+        return render_to_response(
+            'app/change_password.html',
+            variables,
+        )
```

Comment:
The second `return` from line 156 is unnecessary, it's already handled by the return on line 152 which is in the `else` block.

We can make this simpler by removing the else, moving everything beneath one indent to the left and removing lines 156 - 160.